test/Android/Attach/TestScript.xml

```diff
@@ -17,7 +17,7 @@
     <Duration Volatile="True">00:00:01.2064573</Duration>
     <Event name="IDebugProgramCreateEvent2" expected="False" />
     <Event name="IDebugBreakpointBoundEvent2" expected="False">
-      <BoundBreakpoint  IgnoreCase="True">282,0 to 282,0 in libattach.so!android_main(android_app*)</BoundBreakpoint>
+      <BoundBreakpoint  IgnoreCase="True" RegEx="^282,0 to 282,0 in libattach.so!(android_main\(android_app\*\)|0x.*)$" />
```

Comment:
>0x. [](start = 112, length = 3)

Rather than `0x.*` how about `[0-9A-Fa-f]{8,16}` with a `0x` prefix if you really want it here (though personally I don't think there should be a `0x` prefix).